src/main/java/core/basesyntax/model/figure/Triangle.java

```diff
@@ -1,6 +1,15 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Triangle extends Figure {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
let's keep this field in Figure class and remove from childs

Reply:
Done